src/vm/field.h

```diff
@@ -85,10 +85,35 @@ class FieldDesc
     LPUTF8 m_debugName;
 #endif
 
+public:
     // Allocated by special heap means, don't construct me
-    FieldDesc() {};
+    FieldDesc() =delete;
+
+#ifndef DACCESS_COMPILE
+    FieldDesc& operator = (const FieldDesc& src)
```

Comment:
However, I'm thinking about moving all fields (except the pointer) to another structure like `SimplyCopyable` to avoid explicit copy of every member field. Is it a good idea?